ui/node_menu.py

```diff
@@ -4,6 +4,8 @@
 from .. tree_info import getSubprogramNetworks
 from .. utils.nodes import getAnimationNodeTrees
 
+mainListDataTypes = ("Object", "Integer", "Float", "Vector", "String")
```

Comment:
can you rename this to mainBaseDataTypes?
base data types are all the types that also have a corresponding list socket.
the list datatypes would be "Integer List", "Object List", etc.